app/src/app/components/sidebar/RecentMapsModal.tsx

```diff
@@ -0,0 +1,157 @@
+import { useMapStore } from "@/app/store/mapStore";
```

Comment:
there's a silent warning passed in the console: `[zustand persist middleware] Unable to update item 'districtr-persistrictr', the given storage is currently unavailable.`. Per the discussion [here](https://github.com/pmndrs/zustand/discussions/1961) it seems it may be a hydration issue, but as far as I can tell it does not affect the saving or recalling of user maps